src/Platform_Core/org/lobobrowser/main/ReuseServer.java

```diff
@@ -133,7 +133,6 @@ public void run() {
               } else if ("GRINDER".equals(command)) {
                 final DataOutputStream dos = new DataOutputStream(s.getOutputStream());
                 if (blankIdx != -1) {
-                  @SuppressWarnings("null")
```

Comment:
You have probably not enabled `Annotation based Null analysis` in your IDE. I use Eclipse with this setting and without the SupressWarnings there is a warning on the following line.